src/modding/ModManager.cs

```diff
@@ -558,7 +850,35 @@ private void UpdateSelectedModInfo()
 
             leftArrow.Disabled = true;
             rightArrow.Disabled = true;
+            moveModUpButton.Disabled = true;
+            moveModDownButton.Disabled = true;
+        }
+
+        errorInfoLabel.Hide();
+    }
+
+    private List<ImageTexture> LoadModPreviewImages(FullModDetails mod)
+    {
+        var returnValue = new List<ImageTexture>();
```

Comment:
```suggestion
        var result = new List<ImageTexture>();
```
please use that kind of name for generic return value of a method.